kidkode/src/main/webapp/script.js

```diff
@@ -96,56 +90,37 @@ function showResults() {
   resultsContainer.innerHTML = output.join("");
 }
 
-
 /*
- * VARIABLES
+ * HTML containers and variables.
  */
 const quizContainer = document.getElementById("quiz");
 const resultsContainer = document.getElementById("results");
 const submitButton = document.getElementById("submit");
-const myQuestions = [
-  {
-    number: 0,
-    question: "Do you like to play videogames?",
-    answers: {
-      yes: {
-        choice: "yes",
-        points: ["gameDev", "animation", "2-dAnimation"],
-        next: 1
-      },
-      no: {
-        choice: "no",
-        points: ["intDesign", "webDesign", "UX"],
-        next: 2
-      }
-    },
-    previous: null
-  },
-  {
-    number: 1,
-    question: "Do you like to create art?",
-    answers: {
-      yes: {
-        choice: "yes",
-        points: ["gameArt", "animation", "3-dAnimation"],
-        next: 3
-      },
-      no: {
-        choice: "no",
-        points: ["gameDev", "conceptArt", "2-dAnimation"],
-        next: 4
-      }
-    },
-    previous: 0 // previous method will have to subtract point that was just added to hashmap
-  }
-];
 
 /*
- * Build the quiz
+ * Obtain quiz questions from the JSON file and build the quiz.
  */
-buildQuiz();
+function startQuiz() {
+  fetch("quizSteps.json")
```

Comment:
We'll just need to remember to eventually change this from the hardcoded file to the appropriate file based on what the user selects!

Reply:
Right, thanks for reminding me! I can put a comment there so we'll remember that!